docs/swagger/spec.yml

```diff
@@ -1283,6 +1283,143 @@ paths:
       responses:
         200:
           description: RSS feed json as a string
+  /exports/:
+    get:
+      summary: Get a list of exports
+      description: |
+        The exprots API endpoint enables searching, listing, and creating new exports.
+      tags:
+        - Imagery
+      parameters:
+        - $ref: '#/parameters/orderingBase'
+        - $ref: '#/parameters/pageSize'
+        - $ref: '#/parameters/page'
+        - $ref: '#/parameters/organization'
+        - $ref: '#/parameters/project'
+        - $ref: '#/parameters/exportStatus'
+      responses:
+        200:
+          description: Paginated list of exports
+          schema:
+            $ref: '#/definitions/ExportPaginated'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: Create an export
+      description: |
+        Create a new export.
+      tags:
+        - Imagery
+      parameters:
+        - name: Export
+          in: body
+          schema:
+            $ref: '#/definitions/ExportCreate'
+      responses:
+        201:
+          description: Export created
+          schema:
+            $ref: '#/definitions/Export'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  /exports/definition:
```

Comment:
I don't think this should go here -- I think instead (and you already have it) `/exports/{uuid}/definition/` covers the use-case we need. That feels like a more natural API. If we need an endpoint to search/list definitions (I don't think we do) it should be something like `/export-definitions/`.